scripts/verify-dashboards-full-ui-e2e.py

```
@@ -0,0 +1,755 @@
+        and "Avg duration by host (ui e2e)" in d.find_element(by.TAG_NAME, "body").text
+        and "INFO" in d.find_element(by.TAG_NAME, "body").text
+        and "ERROR" in d.find_element(by.TAG_NAME, "body").text
+        and "host-a" in d.find_element(by.TAG_NAME, "body").text
+        and "host-b" in d.find_element(by.TAG_NAME, "body").text
+    )
+    text = driver.find_element(by.TAG_NAME, "body").text  # type: ignore[attr-defined]
+    if "No results found" in text:
+        raise RuntimeError("Dashboard UI still showed 'No results found'")
+
+
+def main() -> int:
```

Ruff (0.15.7)

[warning] Too many statements (57 > 50)

(PLR0915)